docs/versions.rst

```diff
@@ -58,6 +58,48 @@ they will be redirected to the **Default version**.
 This defaults to **latest**,
 but could also point to your latest released version.
 
+States
+------
+
+States define the visibility of a version across the site.
+You can change the states of a version from the :guilabel:`Versions` tab of your project.
+
+Active
+~~~~~~
+
+- **Active**
```

Comment:
I think we can use a definition list here:


```rst
:Active:
  - Docs...
  - Builds...

:Inactive:
  - ...
```

Reply:
Looks weird like that

![Screenshot_2020-04-20 Versions — Read the Docs 4 1 5 documentation](https://user-images.githubusercontent.com/4975310/79800570-10ab2600-8322-11ea-9c82-5aa640ee4aed.png)